chat.js

```diff
@@ -618,6 +653,15 @@ class CommandContext {
 			}
 
 			if (room) {
+				if (room.r9k && !room.isMuted(user)) {
+					let muteTime = this.checkR9k(user, room, message);
+					if (muteTime) {
+						room.mute(user, muteTime * 1000);
+						this.errorReply(`Since Robot9000 mode in enabled in this room and the message you entered was not unique, you've been muted for ${muteTime} seconds.`);
```

Comment:
This sentence does not make gramatical sense.

Maybe it should say: ``Since Robot9000 mode **is** enabled in this room and the message you entered was not unique...``